.github/workflows/publish-npm.yml

```diff
@@ -43,6 +43,12 @@ jobs:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.GRADIO_PAT }}
+      - name: add label to skip chromatic build
+        if: ${{ steps.changesets.outputs.pullRequestNumber != '' && steps.changesets.outputs.pullRequestNumber != 'undefined' }}
```

Comment:
What I said doesn't really matter in this case since this only runs when we merge to main (push), so permissions aren't an issue.  But this is the correct condition to are sure we don't try to label a PR that doesn't exist!